### PR TITLE
test.py: bump timeout of async requests for topology

### PR DESCRIPTION
**Pull request by alecco**
Topology tests do async requests using the Python driver. The driver's
API for async doesn't use the session timeout.

Pass 60 seconds timeout (default is 10) to match the session's.

Fixes https://github.com/scylladb/scylladb/issues/11289

### CODE REVIEW

**Comment by alecco** [comment]
@slivne here

**Review by nyh — APPROVED** [review]
Looks good. Is this wrapper is the only way where test/toplogy tests may ever call execute_async()?
Do we have additional test suites that have this async code besides test/topology?

**Comment by nyh** [comment]
Also, can you please look into the max_schema_agreement_wait parameter (see https://docs.datastax.com/en/developer/python-driver/3.23/api/cassandra/cluster/)? I think we need to increase this too as evident by the warnings we saw.

**Comment by alecco** [comment]
> Looks good. Is this wrapper is the only way where test/toplogy tests may ever call execute_async()?
> Do we have additional test suites that have this async code besides test/topology?

Only `test/topology`

**Comment by alecco** [comment]
> Also, can you please look into the max_schema_agreement_wait parameter (see https://docs.datastax.com/en/developer/python-driver/3.23/api/cassandra/cluster/)? I think we need to increase this too as evident by the warnings we saw.

Added in latest.

**Review by nyh — APPROVED** [review]
Looks good. I'll wait for the CI success to merge.

Please add "Refs #11289" or even Fixes if you are optimistic ;-)

File: test/topology/conftest.py [hunk 1]
```diff
@@ -63,7 +63,7 @@ def on_error(exception, *_):
 
 
 def run_async(self, *args, **kwargs) -> asyncio.Future:
-    return _wrap_future(self.execute_async(*args, **kwargs))
+    return _wrap_future(self.execute_async(*args, **kwargs, timeout=60.0))
```
Comment (fruch) [inline test/topology/conftest.py:66]: I would do that with a bit more care, you are new blocking and call from passing it's own timeout

Check the kwargs first and then if it doesn't have timeout, put the default
```python
kwargs.setdefault("timeout", 60.0)
```

Comment (alecco) [reply test/topology/conftest.py]: Done in latest

**Comment by nyh** [comment]
@alecco I merged this PR, but @fruch suggested in #11289 that maybe there's something we still don't understand, because the ExecutionProfile defined in test/topology/conftest.py rets "request_timeout" and that should have been (according to @fruch 's investigation) enough. Anyway, if you later discover something new about this, you can create a new patch on top of this one.

**Pull request closed without merging**